package.json

```diff
@@ -28,6 +28,7 @@
     "test:lint": "eslint --ext .js,jsx,json ./src ./data ./cypress ./.storybook ./webpack** && swagger-cli validate data/schema.yaml && stylelint 'src/**/*.js' 'src/**/*.jsx'",
     "test:puppeteer": "jest --env=jsdom --colors ./puppeteer",
     "test:unit": "jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"puppeteer\"",
+    "test:unit:watch": "jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"puppeteer\" --watch",
```

Comment:
can this not be `npm run test:unit -- --watch`?

Reply:
Accidentally left this in 🙈 It's so useful to development though. It's one of the first changes I make to a new branch